site.py

```diff
@@ -2,54 +2,61 @@
 """
 
 import sys
-if sys.version_info[0] <3 :
-    raise RuntimeError('We love Python 3.')
-
 import os
-
 from flask import Flask, render_template, url_for, Blueprint
-from flask import redirect, abort
+from flask import abort
 from flask_frozen import Freezer
-import yaml
-
 from elsa import cli
 
+if sys.version_info[0] < 3:
+    raise RuntimeError('We love Python 3.')
+
 app = Flask('naucsepythoncz', template_folder="")
 app.config['TEMPLATES_AUTO_RELOAD'] = True
 
-COURSES = os.listdir(app.root_path+"/courses/")
+COURSES = os.listdir(app.root_path + "/courses/")
 
 # Hacky blueprints for static directories
 
 for course in COURSES:
-    blueprint = Blueprint('course_'+course, __name__,
-        static_url_path='/courses/'+course+'/static',
-        static_folder=app.root_path+"/courses/"+course+"/static")
+    blueprint = Blueprint('course_' + course, __name__,
+        static_url_path='/courses/' + course + '/static',
+        static_folder=app.root_path + "/courses/" + course + "/static")
     app.register_blueprint(blueprint)
 
 
 def course_url(path):
     path = path.rstrip('/')
     return url_for('course', path=path)
 
+
 app.jinja_env.globals['course_url'] = course_url
 
+
 @app.route('/')
 def index():
     return render_template("templates/index.html")
 
+
 @app.route('/courses/<path:path>/')
 def course(path):
-    template_paths = ["courses/"+path+'.html', "courses/"+path+'/index.html']
+    template_paths = ["courses/" + path + '.html', "courses/" + path + '/index.html']
```

Comment:
That's not a very good reason to change code...
Anyway, after the edit the line exceeds [PEP 8 recommendation on line length](https://www.python.org/dev/peps/pep-0008/#maximum-line-length). Could you wrap it, or not edit it?